core/core_modules/kpm.js

```diff
@@ -44,20 +49,34 @@ var git = require.once('../git.js'),
 
                 for (var i = 0; i < args.length; i++) {
                     var url = args[i];
-                    if (!url.startsWith('http') && !url.startsWith('ssh')) {
-                        var spl = url.split('/');
-                        if (spl.length != 2) {
-                            api.sendMessage('Invalid github reference provided "' + url + '". Skipping...', event.thread_id);
-                            continue;
-                        }
-                        url = 'https://github.com/' + url;
+                    if (url.startsWith('http') || url.startsWith('ssh')) {
+                        install.call(this, url, api, event);
+                        continue;
+                    }
+
+                    var spl = url.split('/');
```

Comment:
Unlikely possibility that the module name may have a forward slash in it which would cause it to be missed.
I don't really think this is a hassle, as the module could still be install manually anyway.